ebuildtester/docker.py

```diff
@@ -148,15 +149,23 @@ def _setup_container(self, docker_image):
             docker = subprocess.Popen(docker_args)
             docker.wait()
 
-    def _create_container(self, docker_image, local_portage, overlays):
+    def _create_container(self, docker_image, portdir, overlays):
         """Create new container."""
 
-        distdir = "{}/distfiles".format(local_portage)
+        cache_dir = user_cache_dir("ebuildtester")
+        os.makedirs(cache_dir, exist_ok=True)
+
+        distdir = "{}/distfiles".format(cache_dir)
         os.makedirs(distdir, exist_ok=True)
 
-        pkgdir = "{}/packages".format(local_portage)
+        pkgdir = "{}/packages".format(cache_dir)
         os.makedirs(pkgdir, exist_ok=True)
 
+        if options.OPTIONS.batch:
+            portdir_opt = "ro"
+        else:
+            portdir_opt = "rw"
```

Comment:
```suggestion
        portdir_opt = "ro" if options.OPTIONS.batch else "rw"
```

Reply:
I thought about that but seemed less readable